packages/db/src/core/utils.ts

```diff
@@ -15,6 +15,11 @@ export function getRemoteDatabaseUrl(): string {
 	return env.ASTRO_STUDIO_REMOTE_DB_URL || 'https://db.services.astro.build';
 }
 
+export function isRemoteStudio(url: string): boolean {
+	const protocol = new URL(url).protocol;
+	return protocol === 'http:' || protocol === 'https:';
+}
```

Comment:
Isn't this a bit naive? I thought you could connect to http(s) databases using LibSQL.

Reply:
I was prioritizing compatibility with any existing internal Studio tooling.

Reply:
Does this mean that to connect to http / https LibSQL instances, you need to do something special specific to Astro? I tried to search for `libsql+https` only but couldn't find any references to anything.

Reply:
If there was a different environment variable for this couldn't we avoid this problem?